app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -31,6 +31,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.upChevonIconClass = 'gem-c-accordion-nav__chevron'
     this.downChevonIconClass = 'gem-c-accordion-nav__chevron--down'
 
+    // component content pulled from translated data attributes
+    this.$module.actions = {}
+    this.$module.actions.showText = this.$module.getAttribute('data-show-text')
+    this.$module.actions.hideText = this.$module.getAttribute('data-hide-text')
+    this.$module.actions.showAllText = this.$module.getAttribute('data-show-all-text')
+    this.$module.actions.hideAllText = this.$module.getAttribute('data-hide-all-text')
+    this.$module.actions.thisSectionVisuallyHidden = this.$module.getAttribute('data-this-section-visually-hidden')
+
     // Indicate that js has worked
```

Comment:
Whilst we're here...

```suggestion
    // Indicate that JavaScript has worked
```